src/libslic3r/GCode.cpp

```diff
@@ -2352,11 +2352,17 @@ void GCode::_do_export(Print& print, GCodeOutputStream &file, ThumbnailsGenerato
         auto probe_dist_y  = std::max(1., m_config.bed_mesh_probe_distance.value.y());
         int  probe_count_x = std::max(3, (int) std::ceil(mesh_bbox.size().x() / probe_dist_x));
         int  probe_count_y = std::max(3, (int) std::ceil(mesh_bbox.size().y() / probe_dist_y));
-        this->placeholder_parser().set("bed_mesh_probe_count", new ConfigOptionInts({probe_count_x, probe_count_y}));
         auto bed_mesh_algo = "bicubic";
-        if (probe_count_x < 4 || probe_count_y < 4) {
+        int min_points = 4; // bicubic needs 4 probe points per axis
+        if (probe_count_x + probe_count_y <= 6) { // lagrange needs up to a total of 6 mesh points
```

Comment:
should it be `probe_count_x * probe_count_y <= 6` instead?

Reply:
Oh yes! I’ll correct it

Reply:
Thing is it will never trigger this algo though if I am reading the code right? As the min probe points is 3 * 3 in the lines above?

Reply:
The max probe count will be 2x3 or 3x2 for Lagrange.
For a typical probe distance say 50mm, it will be 100x150(or 150x100), pretty common actually.